### PR TITLE
studyDetails breadcrumb fix

### PR DESCRIPTION
**Pull request by ty-esi**

### CODE REVIEW

File: src/components/Breadcrumb/BreadcrumbView.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import React from "react";
-import { withStyles } from "@material-ui/core";
-import { Link } from "react-router-dom";
+import React from 'react';
+import { withStyles } from '@material-ui/core';
```
Comment (rana22) [inline src/components/Breadcrumb/BreadcrumbView.jsx:2]: Lets Migrate to MUI 5+

File: src/pages/landing/landingView.js [hunk 2]
```diff
@@ -269,7 +269,33 @@ const LandingView = ({
                 className={classes.carouselTabs}
               >
                 <Tab
-                  styleClasses={classes}
+                  styleClasses={{
```
Comment (rana22) [inline src/pages/landing/landingView.js:272]: Lets avoid using inline styles

File: src/pages/study/studyDetailController.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { useQuery } from '@apollo/client';
+import CircularProgress from '@material-ui/core/CircularProgress';
```
Comment (rana22) [inline src/pages/study/studyDetailController.tsx]: Lets migrate to MUI 5+

**Review by rana22 — CHANGES_REQUESTED** [review]

File: src/pages/landing/landingView.js (hunk 2 above)
Comment (ty-esi) [reply src/pages/landing/landingView.js:272]: This is not an inline style. We are passing custom styles to this component. This was the pattern that was already there

File: src/pages/study/studyDetailController.tsx (hunk 3 above)
Comment (ty-esi) [reply src/pages/study/studyDetailController.tsx]: Sure

File: src/components/Breadcrumb/BreadcrumbView.jsx (hunk 1 above)
Comment (ty-esi) [reply src/components/Breadcrumb/BreadcrumbView.jsx:2]: I have a different PR where I am working on the nreadCrumb. I will make that update there

File: src/components/Breadcrumb/BreadcrumbView.jsx [hunk 4]
```diff
@@ -9,47 +9,52 @@ const CustomBreadcrumb = ({ classes, data }) => (
...
           );
         } else {
-          acc.push(
-            <span className={classes.headerNavLink}>{current.name}</span>,
-          );
+          acc.push(<div className={classes.headerNavLink}>{current.name}</div>);
         }
         if (index < data.length - 1) {
-          acc.push(<div style={{ fontSize: "15px" }}>{">"}</div>);
+          acc.push(<div style={{ fontSize: '15px' }}>{'>'}</div>);
```
Comment (rana22) [inline src/components/Breadcrumb/BreadcrumbView.jsx:25]: Lets remove this as well.

**Review by rana22 — APPROVED** [review]
LGTM

**Pull request merged**